clientv3/op.go

```diff
@@ -558,3 +558,6 @@ func isWithPrefix(opts []OpOption) bool { return isOpFuncCalled("WithPrefix", op
 
 // isWithFromKey returns true if WithFromKey is being called in the op
 func isWithFromKey(opts []OpOption) bool { return isOpFuncCalled("WithFromKey", opts) }
+
+// IsWithFromKey returns true if WithFromKey is being called in the op
```

Comment:
@cfc4n great, thanks for the fix! I would suggest to rename the IsWithFromKey to IsWithFromKey instead of creating a new function considering it's only used in the op.go, if that sounds good.